### DISCUSSION TITLE
Send to Public with no Archive

### DISCUSSION THREAD

**Discussion by jdhitsolutions** [buttondown/discussions#34] [General]
I would love an option to send to public, or really any group for that matter, and NOT have the email archived. There's an Archive only option. I would like the opposite. This would allow me to send an informational email or something with perishable information where archiving it would serve no purpose.

**Comment by jmduke** [comment]
We've got a doc describing how to do exactly this!

https://docs.buttondown.email/can-i-exclude-an-email-from-archive

**Comment by jdhitsolutions** [comment]
Don't I feel silly now that I see it in front of my face.

**Reply by jmduke** [reply]
If it's any consolation, the doc is all of four days old so it's entirely possible when you were looking for it I hadn't published it yet :D
